tests/unit/src/test/scala/tests/DefinitionSlowSuite.scala

```diff
@@ -123,9 +123,9 @@ object DefinitionSlowSuite extends BaseSlowSuite("definition") {
            |import java.util.concurrent.Future/*Future.java*/ // unused
            |import scala.util.Failure/*Try.scala*/ // unused
            |import org.scalatest.FunSuite/*FunSuite.scala*/
-           |object MainSuite/*L6*/ extends FunSuite/*FunSuite.scala*/ {
+           |object MainSuite/*<no symbol>*/ extends FunSuite/*FunSuite.scala*/ {
            |  test/*FunSuiteLike.scala*/(testName/*<no symbol>*/) {
-           |    val condition/*L8*/ = Main/*Main.scala:5*/.message/*<no symbol>*/.contains/*String.java*/("Hello")
+           |    val condition/*L9*/ = Main/*Main.scala:5*/.message/*<no symbol>*/.contains/*String.java*/("Hello")
            |    assert/*Assertions.scala*/(condition/*L8*/)
            |  }
            |}
```

Comment:
I tweaked this test case for fixing some that are related to the change, but this test case is still failing (even in `master` branch)...

```
=======
=> Diff
=======
---∙
+++∙
@@ -2,3 +2,3 @@
 package a
-import java.util.concurrent.Future/*<no symbol>*/ // unused
+import java.util.concurrent.Future/*Future.java*/ // unused
 import scala.util.Failure/*Try.scala*/ // unused
@@ -6,3 +6,3 @@
   val message/*L4*/ = Message/*Message.java:1*/.message/*Message.java:2*/
-  new java.io.PrintStream/*<no symbol>*/(new java.io.ByteArrayOutputStream/*<no symbol>*/())
+  new java.io.PrintStream/*PrintStream.java*/(new java.io.ByteArrayOutputStream/*ByteArrayOutputStream.java*/())
   println/*Predef.scala*/(message/*L4*/)
@@ -11,3 +11,3 @@
 package a
-import java.util.concurrent.Future/*<no symbol>*/ // unused
+import java.util.concurrent.Future/*Future.java*/ // unused
 import scala.util.Failure/*Try.scala*/ // unused
@@ -16,3 +16,3 @@
   test/*FunSuiteLike.scala*/("a") {
-    val condition/*L6*/ = Main/*Main.scala:3*/.message/*Main.scala:4*/.contains/*<no symbol>*/("Hello")
+    val condition/*L6*/ = Main/*Main.scala:3*/.message/*Main.scala:4*/.contains/*String.java*/("Hello")
     assert/*Assertions.scala*/(condition/*L6*/)
  tests.DiffAssertions$.colored(DiffAssertions.scala:103)
  tests.BaseSuite.assertNoDiff(BaseSuite.scala:102)
  tests.DefinitionSlowSuite$.$anonfun$new$5(DefinitionSlowSuite.scala:71)
  scala.util.Success.$anonfun$map$1(Try.scala:255)
  scala.util.Success.map(Try.scala:213)
  scala.concurrent.Future.$anonfun$map$1(Future.scala:292)
  scala.concurrent.impl.Promise.liftedTree1$1(Promise.scala:33)
  scala.concurrent.impl.Promise.$anonfun$transform$1(Promise.scala:33)
  scala.concurrent.impl.CallbackRunnable.run(Promise.scala:64)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
```

I'm not sure why we cannot resolve the location of java resources :thinking:

Reply:
Ah, this was just the problem on my local computer, `sudo apt install openjdk-8-source` fixed this :)

Reply:
Our docs should probably mention this for ubuntu users 🤔